openmp/libomptarget/include/device.h

```diff
@@ -159,7 +159,7 @@ struct DeviceTy {
   /// }
 
   /// Register \p Entry as an offload entry that is avalable on this device.
-  void addOffloadEntry(OffloadEntryTy &Entry);
+  void addOffloadEntry(OffloadEntryTy Entry);
```

Comment:
It's only 2 ptrs now but why would we not want (const) references here?

Reply:
With the current usage, it's passed in as an r-value and then stored inside of some other struct. I wanted explicitly ownership by forcing the copy. It could probably be done another way.

Reply:
But it is live and unmodified during the call. I don't see why the copy helps. const ref seems better, especially since we should not store the reference as the entry could be invalid. A copy hides this, I'd say.

Reply:
It's fine as long as it's ultimately copied by the `insert` call, otherwise it's a reference to a temporary. I can make the change.